test/examples/if.ml

```diff
@@ -29,14 +29,21 @@ let t7 = if x then false else y
 
 (* if x then y else true *)
 let t8 = if x then y else true
-    
+
 (* Should ding them twice - encouragement to fix *)
 let double_points = if x then x else true
 
 (* Slightly more complex examples *)
 
 let t9 = if 3 > 0 then 3 > 0 else false
 
+(* comparing boolean literals *)
+(* x = true --> x, y = false ---> not y *)
+let t10 = t1 = true 
+let t11 = t2 = false
+let t12 = true = t1
+let t13 = false = t2
+
```

Comment:
Can you put these lines in a different testing file? This file is exclusively for style involving `if` statements.

Reply:
Whoops, got it. Seems like ``equality.ml`` is more appropriate for this test.